cmd/cadvisor.go

```diff
@@ -140,9 +141,27 @@ func (ml *metricSetValue) Set(value string) error {
 	return nil
 }
 
+type DenyList []string
+
+func (ms *DenyList) String() string {
+	return strings.Join(*ms, ",")
+}
+
+// Set converts a comma-separated string of metrics into a slice and appends it to the DenyList.
+func (ms *DenyList) Set(value string) error {
+	metrics := strings.Split(value, ",")
+	for _, metric := range metrics {
+		metric = strings.TrimSpace(metric)
+		if len(metric) != 0 {
+			*ms = append(*ms, metric)
+		}
+	}
+	return nil
+}
+
 func init() {
 	flag.Var(&ignoreMetrics, "disable_metrics", "comma-separated list of `metrics` to be disabled. Options are 'accelerator', 'cpu_topology','disk', 'diskIO', 'memory_numa', 'network', 'tcp', 'udp', 'percpu', 'sched', 'process', 'hugetlb', 'referenced_memory', 'resctrl'.")
-
+	flag.Var(&ignoreSpecificMetrics, "disable_specific_metrics", "Comma-separated list of metrics not to be enabled. This list comprises of exact metric names and/or regex patterns. ")
```

Comment:
It might be good clarify in the description for the flag on difference between `disable_metrics` and `disable_specific_metrics`